src/DataExplorer/DataExplorerVisualizations.jsx

```diff
@@ -67,12 +91,18 @@ DataExplorerVisualizations.propTypes = {
   arrangerData: PropTypes.object,
   arrangerConfig: PropTypes.object,
   sqon: PropTypes.object,
+  selectedTableRows: PropTypes.array,
+  projectId: PropTypes.string,
+  api: PropTypes.func,
 };
 
 DataExplorerVisualizations.defaultProps = {
   arrangerData: null,
   arrangerConfig: {},
   sqon: null,
+  selectedTableRows: [],
```

Comment:
aren't these props arranger passes to us? I don't think we should define the default value if they are

Reply:
eslint asks for this..

Reply:
I think I would make them `.isRequired` then, arranger always passes them right? Idk I just feel weird setting the default values on props that might not actually be the default values you know?

Reply:
Sure, I'll make them `.isRequired`.